### ISSUE TITLE
possible missing /po/extra/_it.po_

### ISSUE THREAD

**Issue by SwampRabbit** [robertsanseries/ciano#123]
**Describe the bug**
Build failure due to possible missing /po/extra/_it.po_.

**Distro (please complete the following information):**
 - OS: MX-Linux
 - Version [19.4 and 21preAlpha2]
    - Basically Debian 10 and 11

**Info do app (please complete the following information):**
 - Version [0.2.4 + git master]
 - Instalation [schroot building .deb with sbuild and pbuilder]

**To Reproduce (If applicable)**
 - Compile the git master should thow the error below
 - I copied _es.po_ and renamed it _it.po_ in /po/extra/, compiling then completes successfully

**Additional context (If applicable)**
```
[4/30] /usr/bin/meson --internal msgfmthelper ../../data/com.github.robertsanseries.ciano.desktop.in data/com.github.robertsanseries.ciano.desktop desktop /build/com.github.robertsanseries.ciano-0.2.4+git20210214/po/extra
FAILED: data/com.github.robertsanseries.ciano.desktop 
/usr/bin/meson --internal msgfmthelper ../../data/com.github.robertsanseries.ciano.desktop.in data/com.github.robertsanseries.ciano.desktop desktop /build/com.github.robertsanseries.ciano-0.2.4+git20210214/po/extra
msgfmt: error while opening "it.po" for reading: No such file or directory
[5/30] /usr/bin/meson --internal msgfmthelper ../../data/com.github.robertsanseries.ciano.appdata.xml.in data/com.github.robertsanseries.ciano.appdata.xml xml /build/com.github.robertsanseries.ciano-0.2.4+git20210214/po/extra
FAILED: data/com.github.robertsanseries.ciano.appdata.xml 
/usr/bin/meson --internal msgfmthelper ../../data/com.github.robertsanseries.ciano.appdata.xml.in data/com.github.robertsanseries.ciano.appdata.xml xml /build/com.github.robertsanseries.ciano-0.2.4+git20210214/po/extra
msgfmt: error while opening "it.po" for reading: No such file or directory
```
@albanobattistella is this something you were working on?

**Comment by robertsanseries** [comment]
resolved problem at commit 71a4a051cba3b6e38d21a24dd71d4f912df2faab

**Comment by SwampRabbit** [comment]
@robertsanseries thank you very much, I'll be updating the package in MX Linux in the next day or two.
Keep up the great work!
